content/learn/accessibility/keyboard-testing/index.md

```diff
@@ -46,3 +46,9 @@ Ensure that navigation with the Tab key meets the following criteria:
 4. Focus remains in the modal and does not return to the page.
 5. When the Tab key places focus on an element that closes the modal, pressing Enter closes the modal.
 6. When the modal closes, focus returns to the element that opened the modal or goes to a logical place based on actions within the modal. Focus does not return the top of the underlying page.
+
+### 4. Increase page zoom to 200%
+
+1. Ensure that all text of the page has increased in size.
```

Comment:
I think "of" should be "on." ... I'd also change "has increased" to "increases."